app/src/org/commcare/dalvik/activities/HomeActivityUIController.java

```diff
@@ -0,0 +1,404 @@
+package org.commcare.dalvik.activities;
+
+import android.annotation.SuppressLint;
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.text.Spannable;
+import android.text.format.DateUtils;
+import android.util.Pair;
+import android.view.View;
+import android.view.ViewTreeObserver;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.etsy.android.grid.StaggeredGridView;
+
+import org.commcare.android.adapters.HomeScreenAdapter;
+import org.commcare.android.database.UserStorageClosedException;
+import org.commcare.android.view.SquareButtonWithNotification;
+import org.commcare.dalvik.R;
+import org.commcare.dalvik.application.CommCareApplication;
+import org.commcare.dalvik.preferences.CommCarePreferences;
+import org.commcare.suite.model.Profile;
+import org.javarosa.core.services.locale.Localization;
+
+import java.text.DateFormat;
+import java.util.Date;
+
+import in.srain.cube.views.GridViewWithHeaderAndFooter;
+
+/**
+ * Created by amstone326 on 9/17/15.
+ */
+public class HomeActivityUIController {
+
+    private final static String UNSENT_FORM_NUMBER_KEY = "unsent-number-limit";
+    private final static String UNSENT_FORM_TIME_KEY = "unsent-time-limit";
+
+    private final CommCareHomeActivity activity;
+    private SquareButtonWithNotification startButton;
+    private SquareButtonWithNotification logoutButton;
+    private SquareButtonWithNotification viewIncompleteFormsButton;
+    private SquareButtonWithNotification syncButton;
+    private SquareButtonWithNotification viewOldFormsButton;
```

Comment:
The use of "Old" is a little confusing, what about `viewSavedFormsButton`? If you agree it would be worth changing the method names associated with this field as well